nd4j/nd4j-backends/nd4j-tests/src/test/java/org/nd4j/linalg/workspace/SpecialWorkspaceTests.java

```diff
@@ -412,7 +412,6 @@ public void testDeleteMappedFile_1() throws Exception {
 
         Nd4j.getWorkspaceManager().destroyAllWorkspacesForCurrentThread();
 
-        Files.delete(tmpFile);
```

Comment:
This should probably stay in place?

Reply:
Files.createTempFile already will delete the file. No need for the duplicate.